### DISCUSSION TITLE
On the issue of pore water pressure distribution

### DISCUSSION THREAD

**Discussion by LongTIme1111** [Anura3D/Anura3D_OpenSource#197] [Q&A]
Dear All,

I'm simulating a rainfall landslide on a shallow slope, and the lower part of the model is bedrock, and I didn't set this part. After I loaded gravity, the distribution of suction was different from what I expected, so why didn't it increase evenly downwards with a vertical bevel? (Penetration simulation has not yet been carried out)

Or maybe something went wrong with my setup

I would appreciate if someone give me a feedback.

Best regards,
Jack
![28cd929753a0f92c3ba534d5b5fa6cf](https://github.com/Anura3D/Anura3D_OpenSource/assets/162860598/f62cee1f-977b-4c4f-accf-b8331d328035)
![650908caccf10ebbfa997d330c0cde7](https://github.com/Anura3D/Anura3D_OpenSource/assets/162860598/5e3a3032-f92c-422f-98af-924247da798c)

**Comment by WinsonHsia** [comment]
I think setting soil surface, phreatic surface, and K0-procedure well may solve the problem
